### ISSUE TITLE
Invalid path in pandare.qcows

### ISSUE THREAD

**Issue by AndrewFasano** [panda-re/panda#1263]
I imagine this is caused by @jamcleod's changes in #1257?

```
$ python3 -m pandare.qcows mips
Run the generic mips PANDA guest interactively with the following command:
~/git/panda/build/mips-softmmu/panda-system-mips -L ~/git/panda/build/mips-softmmu/pc-bios -os linux-32-debian:3.2.0-4-4kc-malta ~/.panda/debian_7.3_mips.qcow -m 1g -M malta -kernel /home/jmcleod/.panda/vmlinux-3.2.0-4-4kc-malta -append root=/dev/sda1 -nographic -loadvm root
```

The `jmcleod` shouldn't be there!

**Comment by AndrewFasano** [comment]
Looks like there a bunch of references to `/home/jmcleod` in that json file now that should also be fixed

**Comment by AndrewFasano** [comment]
Whoops, this was already fixed in #1262. Thanks Jordan!
